include/librealsense2/h/rs_types.h

```diff
@@ -67,6 +67,31 @@ typedef struct rs2_intrinsics
     float         coeffs[5]; /**< Distortion coefficients */
 } rs2_intrinsics;
 
+/** \brief Video DSM (Digital Sync Module) parameters for calibration (same layout as in FW ac_depth_params)
+    This is the block in MC that converts angles to dimensionless integers reported to MA (using "DSM coefficients").
+*/
+typedef struct rs2_dsm_params
+{
+    unsigned long long timestamp;   /**< system_clock::time_point::time_since_epoch().count() */
+    unsigned short version;         /**< MAJOR<<12 | MINOR<<4 | PATCH */
+    unsigned char model;            /**< rs2_dsm_correction_model */
+    unsigned char flags[5];         /**< TBD, now 0s */
+    float         h_scale;          /**< the scale factor to horizontal DSM scale thermal results */
+    float         v_scale;          /**< the scale factor to vertical DSM scale thermal results */
+    float         h_offset;         /**< the offset to horizontal DSM offset thermal results */
+    float         v_offset;         /**< the offset to vertical DSM offset thermal results */
+    float         rtd_offset;       /**< the offset to the Round-Trip-Distance delay thermal results */
+    unsigned char temp_x2;          /**< the temperature recorded times 2 (ldd for depth; hum for rgb) */
```

Comment:
What are the units here, can it be negative?

Reply:
No clue what happens at negative temps -- it's undefined in the spec at this time and probably won't work